packages/components/src/__future__/ExampleComponent/docs/ExampleComponent.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+import { ComponentMeta, ComponentStory } from "@storybook/react"
+import { ExampleComponent } from "../ExampleComponent"
+
+export default {
+  title: "AIO/ExampleComponent (Do Not Use)",
```

Comment:
I think this will conflict with the other story since the title dictates the path and it's exactly the same